From fdca3b9bbcac7bfa03c9392bde216a7da5b567e2 Mon Sep 17 00:00:00 2001
From: Alex Naidis <alex.naidis@linux.com>
Date: Tue, 4 Oct 2016 17:13:15 +0200
Subject: [PATCH 3/6] skia: correct call for SK_RELEASE

In Skia "NDEBUG" shouldn't be directly
used as conditional. Instead a logic is
present which converts "NDEBUG" into the
Skia internal "SK_RELEASE" macro.

Correct the check in SkTextureCompressor_Blitter
to reflect that.

Change-Id: Ibaac479c1ab5b444543e2a3657552a756d0055d5
Signed-off-by: Alex Naidis <alex.naidis@linux.com>
---
 src/utils/SkTextureCompressor_Blitter.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/utils/SkTextureCompressor_Blitter.h b/src/utils/SkTextureCompressor_Blitter.h
index 0c259d6..742f86d 100644
--- a/src/utils/SkTextureCompressor_Blitter.h
+++ b/src/utils/SkTextureCompressor_Blitter.h
@@ -458,7 +458,7 @@ private:
             return;
         }
 
-#ifndef NDEBUG
+#ifndef SK_RELEASE
         // Make sure that if we have any runs, they all match
         for (int i = 1; i < fNextRun; ++i) {
             SkASSERT(fBufferedRuns[i].fY == fBufferedRuns[i-1].fY + 1);
-- 
2.10.0

